src/agent/script/win64/libfuzzer-coverage/DumpCounters.js

```diff
@@ -83,7 +83,14 @@ function processModule(module, results_dir) {
     return true;
 }
 
-function dumpCounters(results_dir, sample_name) {
```

Comment:
Note, `sample_name` was not provided by any of the users of this script and was only used to pass to a function that didn't have it as an argument.